### PR TITLE
Improve grammar in OpenType fonts guide

### PR DESCRIPTION
**Pull request by digi-booster**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html [hunk 1]
```diff
@@ -151,9 +151,9 @@ <h3 id="Font_variant_shorthand_Cssxreffont-variant">Font variant shorthand ({{Cs
...
+<p>{{cssxref("font-feature-settings")}} is the 'low level syntax' that allows explicit access to every named available OpenType feature. This gives a lot of control but has some disadvantages in how it impacts inheritance and — as mentioned above — if you wish to change one setting, you have to redeclare the entire string (unless you're using <a href="/en-US/docs/Web/CSS/Using_CSS_variables">CSS custom properties</a> to set the values). Because of this, it's best to use the standard properties shown above wherever possible.</p>
 
-<p>There a huge number of possible features. You can see examples of a number of them above, and there are several resources available for finding more of them.</p>
+<p>There are a huge number of possible features. You can see examples of a number of the above, and there are several resources available for finding more of them.</p>
```
Comment (teoli2003) [inline files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html:156]: them sounds corrrect to me. Why the _the?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thanks, there are a few fixes that are needed.

File: files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html:156]: Agree, though perhaps "examples **for** a number of them"

Comment (hv48067) [reply files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html:156]: The word <b>them</b> doesn’t seem to fit this context. Consider replacing it with the.

File: files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
...
+<p>This is sometimes the trickiest thing to work out if you don't have any documentation that came with the fonts (many types of designers and foundries will provide sample pages and CSS just for this very reason). But there are some sites that make it easier to figure out. You can visit <a href="https://wakamaifondue.com">wakamaifondue.com</a>, drop your font file on the circle where instructed, and in a few moments you'll have a full report on all the capabilities and features of your font. <a href="https://axis-praxis.org">Axis-praxis.org</a> also offers a similar capability, with the ability to click on the features to turn them on or off in a given text block.</p>
```
[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html:26]: Please revert to this:
```suggestion
<p>This is sometimes the trickiest thing to work out if you don't have any documentation that came with the fonts (many type designers and foundries will provide sample pages and CSS just for this very reason). But there are some sites that make it easier to figure out. You can visit <a href="https://wakamaifondue.com">wakamaifondue.com</a>, drop your font file on the circle where instructed, and in a few moments you'll have a full report on all the capabilities and features of your font. <a href="https://axis-praxis.org">Axis-praxis.org</a> also offers a similar capability, with the ability to click on the features to turn them on or off in a given text block.</p>
```

File: files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html:156]: Sounds grammatically correct to both Hamish and me…
```suggestion
<p>There are a huge number of possible features. You can see examples of a number of them above, and there are several resources available for finding more of them.</p>
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
If you accept the two suggestions (click the commit button below each of them), we are good to go.

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks

**Pull request merged**